Nimble Templates/App Base.xctemplate/TemplateInfo.plist

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>Kind</key>
+    <string>Xcode.Xcode3.ProjectTemplateUnitKind</string>
+    <key>Identifier</key>
+    <string>co.nimblehq.dt.unit.applicationBase</string>
+    <key>Ancestors</key>
+    <array>
+        <string>co.nimblehq.dt.unit.bundleBase</string>
+    </array>
+    <key>Targets</key>
+    <array>
+        <dict>
+            <key>ProductType</key>
+            <string>com.apple.product-type.application</string>
+            <key>BuildPhases</key>
+            <array>
+                <dict>
+                    <key>Class</key>
+                    <string>Sources</string>
+                </dict>
+                <dict>
+                    <key>Class</key>
+                    <string>Frameworks</string>
+                </dict>
+                <dict>
+                    <key>Class</key>
+                    <string>Resources</string>
+                </dict>
+            </array>
+        </dict>
+    </array>
+    <key>Options</key>
+    <array>
+        <dict>
+            <key>Identifier</key>
+            <string>languageChoice</string>
+            <key>Units</key>
+            <dict>
+                <key>Swift</key>
+                <dict>
+                    <key>Definitions</key>
+                    <dict>
+                        <key>main.swift</key>
+                        <dict>
+                            <key>SortOrder</key>
+                            <integer>999</integer>
+                        </dict>
+                        <key>main.swift:main</key>
+                        <dict>
+                            <key>Beginning</key>
+                            <string/>
+                            <key>End</key>
+                            <string/>
+                        </dict>
+                    </dict>
+                </dict>
+            </dict>
+        </dict>
+    </array>
+    <key>Definitions</key>
+    <dict>
+        <!-- This is defined globally for templates that don't want language choice -->
+        <key>main.m</key>
```

Comment:
This does not generate an extra main.m file right. 👍

Reply:
it based on your language choice. Basically, I just copy it from base template and change ancestor of it to our stuff, it still the same with things we used before except the ancestor.
![Screen Shot 2020-12-21 at 15 48 32](https://user-images.githubusercontent.com/22606906/102757491-fdb28f00-43a3-11eb-9878-22ee26919d78.png)

You can check implementation of this file in `co.nimblehq.dt.unit.cocoaTouchApplicationBase`'s ancestor and compare with old one.

I'm trying to override `PRODUCT_BUNDLE_IDENTIFIER` but still struggle with it and came with this simple solution, maybe there will be an improvement in future.